crates/nassun/src/error.rs

```diff
@@ -1,7 +1,7 @@
 use std::path::PathBuf;
 
 use miette::Diagnostic;
-use node_semver::Version;
+use node_semver::{Version};
```

Comment:
I think you need to run `cargo fmt --all`

Reply:
At some point during this lifetime or the next will I understand why VS Code does the formatting and sometimes doesn't...
Fixed.